pytype/typegraph/solver.cc

```diff
@@ -184,7 +184,7 @@ std::deque<const CFGNode*> PathFinder::FindShortestPathToNode(
 
 const CFGNode* PathFinder::FindHighestReachableWeight(
     const CFGNode* start,
-    std::unordered_set<const CFGNode*, CFGNodePtrHash> seen,
+    std::unordered_set<const CFGNode*, CFGNodePtrHash>& seen,
```

Comment:
I think that Google coding standards would require passing `seen` as a *pointer* (not reference) because of the `seen.insert(node)` at line 207 (which would become `seen->insert(node)`). In general, pass-by-reference should have a `const`.

[Python, of course, passes sets by reference, which often ends up confusing beginners]